### PR TITLE
PR from fork/1180

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR is to merge a branch on a fork of our repo into develop.
- The original PR is https://github.com/LLNL/RAJA/pull/1180
- Please review and approve this one.

**IMPORTANT: Merge this PR after https://github.com/LLNL/RAJA/pull/1185 is merged**

### CODE REVIEW

File: include/RAJA/policy/sycl/kernel/SyclKernel.hpp [hunk 1]
```diff
@@ -126,25 +108,18 @@ void SyclKernelLauncher(Data data, cl::sycl::nd_item<3> item)
  * Helper class that handles SYCL kernel launching, and computing
  * maximum number of threads/blocks
  */
-template<typename LaunchPolicy, typename StmtList, typename Data, typename Types>
+template<bool, typename LaunchPolicy, typename StmtList, typename Data, typename Types>
```
Comment (trws) [inline include/RAJA/policy/sycl/kernel/SyclKernel.hpp:111]: @homerdin, I'd prefer to have a name here for the bool so we have a trail later, would you mind a change like this?

```suggestion
template<bool IsTriviallyCopyable, typename LaunchPolicy, typename StmtList, typename Data, typename Types>
```

**Comment by trws** [comment]
Pretty sure we have to wait this one on a merge of the other, then see if we need to rebase it, but otherwise I'm good.

File: include/RAJA/policy/sycl/kernel/SyclKernel.hpp (hunk 1 above)
Comment (rhornung67) [reply include/RAJA/policy/sycl/kernel/SyclKernel.hpp:111]: Good suggestion. 👍

**Comment by rhornung67** [comment]
Folks, this should be good to merge when it comes back all green. Please review and approve or make comments or suggestions. Thank you.

**Review by rchen20 — APPROVED** [review]

**Review by homerdin — APPROVED** [review]

**Comment by trws** [comment]
I'm good with this, but would feel more comfortable if we has a test or example or something using it in the near future.

**Review by trws — APPROVED** [review]

**Comment by rhornung67** [comment]
> I'm good with this, but would feel more comfortable if we has a test or example or something using it in the near future.

I'm going to start adding some tests soon. It would also be good to get @homerdin involved in some of this when he gets his LLNL org access. @homerdin are you up for it?

**Comment by homerdin** [comment]
@rhornung67 Yeah, definitely.  Let me know how I can help.

**Review by artv3 — APPROVED** [review]

**Comment by trws** [comment]
He should be in the org, or at least have the invite, now.

**Pull request merged**